app/src/main/res/layout/activity_parking_reservation.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/main_background">
+
+    <TextView
+        android:id="@+id/text_dialog_fragment_parking_size"
+        style="@style/TextStyleAllAppTitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_margin="@dimen/margin_all_general_normal"
+        android:background="@drawable/rounded_edges"
+        android:text="@string/title_activity_parking_reservation"
+        android:textAlignment="center"
+        android:textColor="@color/white"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <EditText
```

Comment:
create a style for the EditText pls you can put there

android:layout_margin
android:autofillHints
android:inputType
android:minHeight
android:textAlignment
android:textColorHint

Reply:
Done, but I wasn't allowed to put autofillhints on style